osbs/schemas/container.json

```diff
@@ -106,6 +106,24 @@
                         "enable-party-popper"
                       ]
                   }
+                },
+                "packages": {
+                  "description": "the packages object to pass to the Cachito request",
+                  "type": ["object", "null"],
+                  "examples": [
+                    {"npm": [{"path": "client"}, {"path": "proxy"}]}
+                  ],
+                  "additionalProperties" : {
```

Comment:
Why isn't this "properties" instead?

We are usually quite restrictive in the container.yaml files in dist-git. So if a user has a typo or is trying to use some invalid key here, we just halt early during validation.

That said, if there is no specific reason for using additional properties here, could we change it to `properties` and set `additionalProperties` to `false`?

Reply:
@athos-ribeiro I used `additionalProperties` so that the schema does not need to be updated every time a new package manager is added to Cachito.